server/app/assets/javascripts/toast.ts

```diff
@@ -32,7 +32,7 @@ export class ToastController {
       'absolute',
       'hidden',
       'left-1/2',
-      'top-0',
+      'top-6',
```

Comment:
This moves the toasts down below the new banner because they were half overlapping the black banner and half over the white background.